lib/omnibus-ctl.rb

```diff
@@ -694,6 +694,11 @@ def run(args)
 
       command_to_run = args[0]
 
+      if command_to_run == "--help"
+        help
+        Kernel.exit 0
+      end
```

Comment:
This is calling the code that the existing help calls (the help method), but this previously didn't know how to handle the flag `--help` vs the command `help`. It would exit 1 because it didn't recognize `--help` as a command.